sdk/tests/deqp/framework/delibs/debase/deMath.js

```diff
@@ -48,6 +48,24 @@ deMath.deInBounds32 = function(a, mn, mx) {
 deMath.deFloatFrac = function(a) { return a - Math.floor(a); };
 
 /**
+ * Global variable used in function deMath.toFloat32()
+ */
+deMath.FLOAT32ARRAY = new Float32Array(1);
+
+/**
+ * Transform a 64-bit float number into a 32-bit float number.
+ * Native dEQP uses 32-bit numbers, so sometimes 64-bit floating numbers in JS should be transformed into 32-bit ones to ensure the correctness of the result.
+ * @param {number} a
+ * @return {number}
+ */
+deMath.toFloat32 = function(a) {
+    deMath.FLOAT32ARRAY[0] = a;
+    return deMath.FLOAT32ARRAY[0];
+};
+
+/** @const */ deMath.INV_LOG_2_FLOAT32 = deMath.toFloat32(1.44269504089); /** 1.0 / log_e(2.0) */
```

Comment:
C++ code:
https://android.googlesource.com/platform/external/deqp/+/master/framework/delibs/debase/deMath.h#36
Treat the 32-bit number 1.44269504089 as a constant